.goreleaser.yml

```diff
@@ -50,14 +50,66 @@ archives:
     <<: *archive_defaults
     wrap_in_directory: false
     format: zip
+aurs:
+  -
+    homepage: "https://github.com/fastly/cli"
+    description: "A CLI for interacting with the Fastly platform"
+    maintainers:
+      - 'oss@fastly.com'
+    license: "Apache license 2.0"
+    skip_upload: auto
+
+    # The SSH private key that should be used to commit to the Git repository.
+    # This can either be a path or the key contents.
+    #
+    # WARNING: do not expose your private key in the config file!
+    private_key: '{{ .Env.AUR_KEY }}'
```

Comment:
I'm not sure where this private key is supposed to come from.

Reply:
I think we just need to generate a new SSH key using `ssh-keygen` similar to how GitHub works. I actually just recently found out that no two GitHub accounts can share an SSH key because of how ssh auth works with git and that's the same here - https://docs.github.com/en/authentication/troubleshooting-ssh/error-key-already-in-use

Reply:
Here are the GitHub docs on generating a key (I never remember how and always have to look at these docs :smile: ) - https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent